locales/en.default.schema.json

```diff
@@ -1801,6 +1870,37 @@
             "label": "Large"
           },
           "info": "For best results, use an image with a 16:9 aspect ratio. [Learn more](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/image-editor#understanding-image-aspect-ratio)"
+        },
+        "desktop_content_alignment": {
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Desktop content alignment"
+        },
+        "header": {
+          "content": "Mobile Layout"
+        },
+        "mobile_content_alignment": {       
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Mobile content alignment"
```

Comment:
Investigating if should become "Content alignment" - see comment https://github.com/Shopify/dawn/pull/900/files#r757633434

Reply:
I think you could do the changes here maybe and request the translations async ? This we don't need to do a follow up 🤷

Reply:
I think we are waiting on @katycobb to see this and comment on it before we make any changes🤔 

https://github.com/Shopify/dawn/pull/900#discussion_r757633434

Reply:
Created a follow up issue https://github.com/Shopify/dawn/issues/959